portfolio/pom.xml

```diff
@@ -38,6 +38,13 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+
+    <!-- Gson dependency -->
```

Comment:
Couldn't figure out how to reply to your comment about this file, sorry.

If it's unused, then you should remove the code since it can help avoid confusion. If you think you might use it in the future, since this is a project only you'll be working on, you can comment out the code and add a comment saying that it's currently unused because you decided not to use gson.